scripts/dump_local_db.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
```

Comment:
After scraping a whole bunch of files into my local db, I wrote this script to make a backup of the database. Seems worth keeping around.